context/app/static/js/components/workspaces/CreateWorkspaceDialog/CreateWorkspaceDialog.jsx

```diff
@@ -46,7 +51,7 @@ function CreateWorkspaceDialog({
             </label>
```

Comment:
This concern is really about the button component on the line above it, but:
 - let's add `disabled={errorMessages.length > 0}` so users can't submit the form if any of the error conditions are met.
 - I really don't like how the label wraps the button and the button is rendered as a span - this is really not great for accessibility and the intended effect is achievable in a semantically correct way.
   - Add an ID to the `form` box above (e.g. `create-workspace-form`)
   - Change the button props to `type="submit" htmlFor="create-workspace-form"`
   - Remove the label wrapping the button and the hidden submit input on line 38

Reply:
This is a really good catch.

Reply:
Not exactly sure what the label was for, removed all those extra lines and still works fine. 

![Uploading image.png…]()

Reply:
Let me know if I am missing something, applied the suggestion, but now it is no longer working: 

![image](https://github.com/hubmapconsortium/portal-ui/assets/45410262/6a85c4e2-4fdc-434f-9ff9-38c0ea2108c0)

Reply:
My mistake - instead of htmlFor on the button, "form=" should work https://www.impressivewebs.com/html5-form-attribute/

Reply:
Works now, thanks :)